rosa_cluster_admin/rosa-cluster-autoscaling.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="rosa-cluster-autoscaling"]
+= Cluster autoscaling
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-cluster-autoscaling
+
+toc::[]
+
+Applying autoscaling to an {product-title} cluster involves configuring a cluster autoscaler and then configuring machine autoscalers for each machine type in your cluster.
```

Comment:
I'm assuming "product-title" here would be ROSA, meaning it shouldn't be "an" rather "a"
```suggestion
Applying autoscaling to a {product-title} cluster ...
```

Reply:
Yeah, this is a side effect of the modularization of docs. Let me see what I can do here, as the original said "OpenShift" and even the OSD one says "an OpenShift", but for ROSA, it does say "an Red Hat..." I'll get back to this one.

Reply:
Fixed this by making clusters plural in the sentence, removing the need for the "a" or "an" and removing that confusion.